### PR TITLE
Fix dataLabels drawing. Migrates to ChromeHeadless

### PR DESCRIPTION
**Pull request by zBritva**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,7 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-  - "7"
-before_install:
-  - sudo apt-get install -y chromium-browser
-  - export CHROME_BIN=chromium-browser
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
+  - "10"
```
Comment (ignatvilesov) [inline .travis.yml]: Please remove lib and .tmp from the repo. Both directories must be in gitignore

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (ignatvilesov) [inline .vscode/settings.json]: Do we need an empty config?

File: package.json [hunk 3]
```diff
@@ -61,7 +61,8 @@
     "tslint": "5.4.3",
     "tslint-microsoft-contrib": "5.0.0",
     "typescript": "2.9.2",
-    "webpack": "^3.8.1"
+    "webpack": "^3.8.1",
```
Comment (ignatvilesov) [inline package.json]: Please note I've also update versions of other package. Please update the same packages as well

File: .vscode/settings.json (hunk 2 above)
Comment (zBritva) [reply .vscode/settings.json]: I removed

**Pull request merged**